javascriptv3/example_code/glacier/README.md

```diff
@@ -1,13 +1,16 @@
 # TypeScript environment for Amazon Glacier examples
-Environment for AWS SDK for JavaScript (V3) AWS Glacier examples. For more information, see the 
-[AWS documentation for these examples](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/glacier-examples.html).
+Environment for AWS SDK for JavaScript (v3) AWS Glacier examples. 
+
```

Comment:
AWS Glacier -> Amazon S3 Glacier (here and throughout).

Reply:
Changed 'AWS [service_name]' to 'Amazon [service_name] throughout